README.md

```diff
@@ -1,30 +1,169 @@
+    # iterates through all your schemata and yields schema_identifier and path to schema folder
+    raise NotImplemented
+
+def schemata() -> cerializer.schema_handler.CerializerSchemata:
+    schemata = []
+	for schema_identifier, schema_root in list_schemata():
+		schema_tuple = schema_identifier, yaml.unsafe_load( # type: ignore
+			open(os.path.join(schema_root, 'schema.yaml'))
+		)
+		schemata.append(schema_tuple)
+	return cerializer.schema_handler.CerializerSchemata(schemata)
+```
+
+2. Create an instance of Cerializer for each of your schemata by calling `cerializer_handler.Cerializer` .
+eg. `cerializer_instance = cerializer_handler.Cerializer(cerializer_schemata, schema_namespace, schema_name)`
+This will create an instance of Cerializer that can serialize and deserialize data in the particular schema format.
+
+3. Use the instance accordingly.
+  eg. 
+  ```python
+ data_record = {
     'name': 'Matej',
     'age': 22
  }
- student_schema_1.serializer(data_record, output)
- print(output.getvalue())
- </code></pre>
+ cerializer_instance = cerializer.cerializer_handler.Cerializer(cerializer_schemata, 'school', 'student')
+ serialized_data = cerializer_instance.serialize(data_record)
+ print(serialized_data)
+```
+
+Serialized data
+```
+b'\x08aaaa\x9c\x02\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x02\xf6\x01\x00'
+```
```

Comment:
It is weird that such a short `data_record` produces so long serialized output. Is it really correct?
```
 data_record = {
    'name': 'Matej',
    'age': 22
 }
```

Reply:
Now it should be fine.